system/Debug/Toolbar/Views/toolbar.js

```diff
@@ -138,6 +155,11 @@ var ciDebugBar = {
         }
     },
 
+    switchClass  : function(el,classFrom,classTo){
+        ciDebugBar.removeClass(el,classFrom);
+        ciDebugBar.addClass(el,classTo);
+    },
+
```

Comment:
Can be replaced with the one-liner: `el.classList.replace()` - not sure if we need this method.

Reply:
@michalsn , Sometimes the class to be replaced doesn't exist, but we still have to add the new class,
If we use classList.replace()  then this doesn't work

ex.
we must add class **dblock** n remove class **ndisplay**
but in some condition the element do not have yet class **ndisplay**

so if we use replace. the function cant work and the **dblock** class will not be added to the element

Reply:
Ok, that's fine. I haven't looked in-depth at what html is generated, so if there is a reason to have the `switchClass` method - go for it.

Reply:
thanks @michalsn